sdk/schemaregistry/Azure.Data.SchemaRegistry/src/Serialization/SchemaRegistrySerializer.cs

```diff
@@ -0,0 +1,560 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+using Azure.Data.SchemaRegistry;
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core.Pipeline;
+using Azure.Messaging;
+using System.Text.Json;
+using Azure.Core.Serialization;
+
+namespace Azure.Data.SchemaRegistry.Serialization
+{
+    /// <summary>
+    /// The Schema Registry serializer serializes and deserializes payloads using the <see cref="SchemaRegistrySerializerOptions.Serializer"/>.
+    /// By default, this is a JSON object serializer. It uses an implemented <see cref="SchemaValidator"/> and a <see cref="SchemaRegistryClient"/>
+    /// to enrich messages inheriting from <see cref="MessageContent"/> with a Schema Id.
+    /// </summary>
+    /// <remarks>
+    /// See <see cref="SchemaValidator"/> for additional information.
+    /// </remarks>
```

Comment:
I'd simplify and just make this a `<seealso>` reference rather than an embedded remark.